kurtosis-module/impl/services/contract_helper.ts

```diff
@@ -7,7 +7,9 @@ import { getPrivateAndPublicUrlsForPortId, ServiceUrl } from "../service_url";
 const SERVICE_ID: ServiceID = "contract-helper-service"
 const PORT_ID = "rest";
 const PORT_NUM: number = 3000;
+const PUBLIC_PORT_NUM: number = 3001;
```

Comment:
Let's make this something high, so it's not liable to collide with anything else they're running on their machine. Maybe